lib/spack/spack/cmd/ci.py

```diff
@@ -167,11 +167,27 @@ def setup_parser(subparser):
         help=spack.cmd.first_line(ci_reproduce.__doc__),
     )
     reproduce.add_argument("job_url", help="Url of job artifacts bundle")
+    reproduce.add_argument(
+        "--runtime",
+        help="Container runtimer to use.",
```

Comment:
```suggestion
        help="Container runtime to use.",
```

Quick typo fix.

Reply:
Otherwise this all looks good to me! I'll defer final merge decision to @scottwittenburg once the typo is fixed.